styles.css

```diff
@@ -264,3 +264,85 @@ button:disabled {
   color: #396df2;
   width: 50%;
 }
+
+/* contact form styling */
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  border-top-left-radius: 80px;
+  background: #6070ff;
+  align-items: center;
+  width: 22vh 8vw;
+}
+
+.contact-me {
+  margin-top: 103px;
+  justify-content: center;
+  font-size: 40px;
+  font-weight: 700;
+  color: #fff;
+}
+
+.contact-form p {
+  font-size: 20px;
+  font-weight: 400;
+  text-align: center;
+  color: #fff;
+}
```

Comment:
- [ ] What do you think of using the color given in the Figma template for `p` to get a design close to given according to the project requirements 📌✨ The color given in Figma template is `#EBEBFF` 😎
- [ ] Also, can you please add a bit of space to the left and right of `p` as given in the design guidelines ? 😄✌

Your design | Given design 
---| ---|
![image](https://github.com/ausiemass/portfolio-austin/assets/106119105/45a7c867-78b8-43d7-be77-ec936294b900) | ![image](https://github.com/ausiemass/portfolio-austin/assets/106119105/a7bccb33-1b54-420f-b005-72d012bd3498)